src/main/java/Lesson7/Plate.java

```diff
@@ -0,0 +1,24 @@
+package Lesson7;
+
+public class Plate {
+    private int food;
+
+    public Plate(int food){
+        this.food = food;
+    }
+    public boolean decreaseFood(int n){
+        if ((food - n) < 0){
+            return false;
+        }else {
+            food -= n;
+            return true;
+        }
+    }
+    public void increaseFood(int n){
+        food += n;
+    }
+    public void Info(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.